### PR TITLE
Use semantic headings in user settings Security

### PR DESCRIPTION
**Pull request by kerryarchibald**
For https://github.com/vector-im/element-web/issues/24922

Screenshot 2023-05-04 at 15 53 50
Screenshot 2023-05-04 at 15 54 00

Screenshot 2023-05-03 at 16 18 59
Screenshot 2023-05-03 at 16 18 27
Screenshot 2023-05-03 at 16 16 59
Screenshot 2023-05-03 at 16 16 30
Screenshot 2023-05-03 at 16 16 01
Screenshot 2023-05-03 at 16 15 31

## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [ ] Linter and other CI checks pass
-   [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Use semantic headings in user settings Security ([\#10774](https://github.com/matrix-org/matrix-react-sdk/pull/10774)). Contributed by @kerryarchibald.

### CODE REVIEW

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx [hunk 1]
```diff
@@ -340,28 +336,24 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
...
-                            </p>
-                            <AccessibleButton kind="link" onClick={onClickAnalyticsLearnMore}>
-                                {_t("Learn more")}
-                            </AccessibleButton>
-                        </div>
+                <SettingsSection heading={_t("Privacy")}>
+                    <SettingsSubsection
+                        heading={_t("Analytics")}
+                        description={_t(
+                            "Share anonymous data to help us identify issues. Nothing personal. " + "No third parties.",
```
Comment (richvdh) [inline src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx:343]:
```suggestion
                            "Share anonymous data to help us identify issues. Nothing personal. No third parties.",
```

File: src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx [hunk 2]
```diff
@@ -373,67 +365,61 @@ export default class SecurityUserSettingsTab extends React.Component<IProps, ISt
...
                         client={client}
                     />
-                </div>
+                </SettingsTab>
             );
         }
 
         return (
-            <div className="mx_SettingsTab mx_SecurityUserSettingsTab">
+            //mx_SecurityUserSettingsTab
```
Comment (richvdh) [inline src/components/views/settings/tabs/user/SecurityUserSettingsTab.tsx:410]: Not really sure what this comment means. Suggest we get rid of it?

```suggestion
```

**Review by richvdh — APPROVED** [review]
seems largely plausible other than the nits below.

@kerryarchibald I think that, in contrast to the PR summary, this contains changes which affect several of the user settings tabs, not just Security. I find reviewing these big PRs really difficult and time-consuming, and would really love it if you could break them down, either into separate PRs, or by rebasing them so that each commit makes a self-contained change.

**Comment by kerryarchibald** [comment]
@richvdh This had its base automatically updated after the labs PR was merged, leading to a bigger diff than intended.
I could have just opened one PR at a time, but I don't want this relatively simple task to take a month.
If the PR looks too big when you come to it, please just flag it!

**Pull request merged**